lib/hmmlearn/hmm.py

```diff
@@ -307,18 +309,218 @@ def _do_mstep(self, stats):
                                      (cvweight + stats['post'][:, None, None]))
 
 
-_MULTINOMIALHMM_DOC_SUFFIX = """
+
+class MultinomialHMM(BaseHMM):
+    """
+    Hidden Markov Model with multinomial emissions
+
+    Attributes
+    ----------
+    n_features : int
+        Number of possible symbols emitted by the model (in the samples).
+
+    monitor_ : ConvergenceMonitor
+        Monitor object used to check the convergence of EM.
+
+    startprob_ : array, shape (n_components, )
+        Initial state occupation distribution.
+
+    transmat_ : array, shape (n_components, n_components)
+        Matrix of transition probabilities between states.
+
+    emissionprob_ : array, shape (n_components, n_features)
+        Probability of emitting a given symbol when in each state.
+
+    Examples
+    --------
+    >>> from hmmlearn.hmm import MultinomialHMM
+    """
+
+    def __init__(self, n_components, n_trials,
```

Comment:
Looking at the other classes, I think it would be consistent to default n_components=1; this also requires n_trials to have a default value which I would suggest to be None, meaning that it will get auto-set in _check_and_set_multinomial_n_features_n_trials, i.e. an expanded version of _check_and_set_multinomial_n_features which also correctly sets n_trials if it is currently None, or errors out if it is set to an incompatible value.

Reply:
Is there any reason you'd prefer the setting of n_trials to be in a separate function _check_and_set_multinomial_n_features_n_trials instead of doing this in _check_and_set_multinomial_n_features ?

Reply:
It's just a matter of renaming the function so that its name matches what it does?  Or am I missing something?

Reply:
Oh, I thought you meant defining another function `_check_and_set_multinomial_n_features_n_trials` in addition to `_check_and_set_multinomial_n_features`